protocol/avdl/keybase1/identify.avdl

```diff
@@ -46,5 +46,15 @@ protocol identify {
    */
   Identify2Res identify2(int sessionID, UID uid, string userAssertion, IdentifyReason reason, boolean useDelegateUI=false, boolean alwaysBlock=false, boolean noErrorOnTrackFailure=false, boolean forceRemoteCheck=false, boolean needProofSet=false, boolean allowEmptySelfID=false, boolean noSkipSelf=true, boolean canSuppressUI=false, TLFIdentifyBehavior identifyBehavior=0, boolean forceDisplay=false);
 
+  record IdentifyLiteRes {
+    UserOrTeamLite ul;
+    union { null, IdentifyTrackBreaks } trackBreaks;
```

Comment:
This is going to be the only `Identify*` call that KBFS uses, even for non-team chats, so I think we need to keep it.